src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerOptions.cs

```diff
@@ -17,6 +17,13 @@ public class ExceptionHandlerOptions
     /// </summary>
     public PathString ExceptionHandlingPath { get; set; }
 
+    /// <summary>
+    /// Gets or sets whether the handler needs to create a separate <see cref="IServiceProvider"/> scope and
+    /// replace it on <see cref="HttpContext"/>.
```

Comment:
Incredibly minor tweak suggestion:

```suggestion
    /// Gets or sets a flag to indicate whether the exception handler should apply a new <see cref="IServiceProvider"/> scope to the <see cref="HttpContext"/>.
```